proposals/scopes.md

```diff
@@ -307,38 +337,35 @@ A|    var _x = 1;
 `LX CY`: Line X Column Y
 
 ```
-Start Scope C0 { // A
-  field flags: has definition
-  info flags:
-  definition: file.js L1 C0 - L6 C17
+Start Generated Range C0 { // A
+  field flags: has definition, is scope
+  definition: file.js, scope offset 0
+  callsite: none
   bindings: x -> _x, z -> _z
 }
 ;
-Start Scope C16 { // B
-  field flags: has name, has definition
-  info flags: function, inherit parent bindings
-  name: z
-  definition: file.js L2 C20 - L5 C1
+Start Generated Range C16 { // B
+  field flags: has definition, is scope
+  definition: file.js, scope offset 1
+  callsite: none
   bindings: message -> _m, y -> _y
 }
 ;
 ;
 ;
-End Scope C1 // B
+End Generated Range C1 // B
 ;
-Start Scope C0 { // C
-  field flags: has name, has definition, has callsite
-  info flags: function, inherit parent bindings
-  name: z
-  definition: file.js L2 C0 - L5 C1
+Start Generated Range C0 { // C
+  field flags: has definition, has callsite
+  definition: file.js, scope offset 1
   callsite: file.js L6 C0
```

Comment:
Even though this PR doesn't change this, but shouldn't the call-site be `L5 C0` (assuming 0-based numbers)?

Reply:
Thanks for catching that, should be updated now.